.gitignore

```diff
@@ -1,6 +1,6 @@
 # Generated by Cargo
 # will have compiled files and executables
-/target/
+**/target/
```

Comment:
This can potentially ignore some valid `target` directories. Maybe it is better to specify full path here

Reply:
Specified it to ignore targets in bob* directories (as well as the one in the root directory)